MonitorizareVot/Onboarding/Controllers/AddNotesViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AddNotesViewController.swift
+//  MonitorizareVot
+//
+//  Created by Geart Otten on 13/10/2019.
+//  Copyright © 2019 Code4Ro. All rights reserved.
+//
+
+import UIKit
+
+class AddNotesViewController: UIViewController {
```

Comment:
Please rename these classes to contain the word `Onboarding` or something similar in their name. These names create confusion because there are controllers that will actually handle this work. Even if those are not named correctly now, they will be in the near future.